src/libsystemd-network/sd-ipv4acd.c

```diff
@@ -60,6 +71,12 @@ struct sd_ipv4acd {
         unsigned n_iteration;
         unsigned n_conflict;
 
+        /* Indicates the maximum duration of the protocol in milliseconds, from the beginning until the first
+         * announcement. Every interval is scaled in a proportional way, except for intervals happening after
+         * the first announcement which keep the original value from the RFC. The actual interval is computed
+         * by multiplying this value for the scaled constants. */
+        uint64_t timeout_multiplier;
```

Comment:
Rather than this, please store the time unit calculated from the specified timeout in usec, i.e. `timeout / 7`.
Please always store time related value in usec.